pandas/core/groupby/generic.py

```diff
@@ -1751,7 +1752,7 @@ def count(self):
         ]
         blk = map(make_block, counted, loc)
```

Comment:
while you're at it, can you rename `blk` to `blks` or something else clearly plural

Reply:
Ok